examples/babi_memnn.py

```diff
@@ -94,8 +94,13 @@ def vectorize_stories(data, word_idx, story_maxlen, query_maxlen):
             pad_sequences(Xq, maxlen=query_maxlen), np.array(Y))
 
 
-path = get_file('babi-tasks-v1-2.tar.gz',
-                origin='http://www.thespermwhale.com/jaseweston/babi/tasks_1-20_v1-2.tar.gz')
+try:
+    path = get_file('babi-tasks-v1-2.tar.gz', origin='http://www.thespermwhale.com/jaseweston/babi/tasks_1-20_v1-2.tar.gz')
+except:
+    print("Error downloading dataset, please download it manually:")
+    print("$ wget http://www.thespermwhale.com/jaseweston/babi/tasks_1-20_v1-2.tar.gz")
```

Comment:
You don't need multiple print statements, a single one will work fine. Also note that `'` is the quote symbol used in this file.